checks/raw/pinned_dependencies.go

```diff
@@ -111,6 +112,27 @@ func collectDockerfileInsecureDownloads(c *checker.CheckRequest, r *checker.Pinn
 	}, validateDockerfileInsecureDownloads, r)
 }
 
+func fileIsInVendorDir(pathfn string) bool {
+	cleanedPath := filepath.Clean(pathfn)
+	splitCleanedPath := strings.Split(cleanedPath, "/")
+
+	// If the project vendors, this is likely to be how
+	// they do it. Therefore, check the first index
+	// before looping.
```

Comment:
I don't follow this comment. What's the need to check `splitCleanedPath[0]` if we're looping anyway?

Reply:
Removed in https://github.com/ossf/scorecard/pull/3675/commits/0310029e6ad492e91545f34f815ea9fe9583e137